### DISCUSSION TITLE
Minimum/Recommended specs?

### DISCUSSION THREAD

**Discussion by DrV0rt3X** [ppy/osu#16183] [Q&A]
this might be a weird thing to ask. 

So I downloaded  osu lazer last week and since downloading it i've been having many frame issues i tried turning my refresh rate down to 60hz instead of 144hz and it was even worse 

So do we have any minimum/recommended specs for running osu lazer?

My specs:
AMD fx 4300 2core (horrible cpu just so everyone knows)
6gigs RAM
Radeon RX 560

**Comment by bdach** [comment]
We don't really have a minimum/required software specification to offer. The eventual goal is to make the game playable on as many computers as it's feasible to do so.

As for your troubles, did you try single-threaded mode at all? With your two-core CPU multithreaded mode may be pretty taxing on the processor as multi-threaded mode uses four threads (so either 100% utilisation if your CPU has hyperthreading, or above quota if not), and the hope would be that single-threaded may be more tolerable.

Difficult to say anything else for sure without logs / a video with the performance overlay showing. A future improvement is getting DirectX compatibility mode online via ANGLE which may also improve the situation.

**Reply by DrV0rt3X** [reply]
i haven't tried using single threaded mode but i'll defenitely give it a shot

**Comment by peppy** [comment]
If your PC can run windows 10/11 then it should be ample to run osu!.

**Reply by DrV0rt3X** [reply]
The thing is my pc has windows 10 it can run OSU just fine. Bust it's OSU Lazer I'm having trouble running  @_@ anyway I'll be upgrading hopefully come new years so that won't be a problem
